pixart.js

```diff
@@ -1 +1,30 @@
+var form = document.getElementById('form');
+var input = document.querySelector('#color-field');
+var brush = document.querySelector('.brush');
 
+//another attempt at getting the color to switch
+/*onclick="document.getElementById('brush').style.color = 'red'">*/
```

Comment:
If you decide to go the `onclick` route, make sure to attach it to the element that needs the event listener (e.g., `document.querySelector( "button" ).onclick = function( ){ ... }`).